client/my-sites/stats/stats-post-likes/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ **/
+import React from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Count from 'components/count';
+import Gridicon from 'components/gridicon';
+import Gravatar from 'components/gravatar';
+import StatsModuleContent from '../stats-module/content-text';
+import QueryPostLikes from 'components/data/query-post-likes';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+import toggleInfo from '../toggle-info';
+import { isRequestingPostLikes, getPostLikes, countPostLikes } from 'state/selectors';
+
+export const PostLikes = ( { countLikes, isRequesting, likes, opened, postId, siteId, toggle, translate } ) => {
+	const infoIcon = opened ? 'info' : 'info-outline';
+	const classes = {
+		'is-showing-info': opened,
```

Comment:
I noticed while the components are loading the `module-header-title` and `infoIcon` are both displayed in this new component.  If we add in a `is-loading` class while the query is still loading then this component will match the behavior of the others on the page.